packages/react-components/src/organisms/__tests__/RichText.test.tsx

```diff
@@ -52,6 +60,24 @@ it('Displays error when styling applied to h2', () => {
   expect(container.textContent).toContain('Invalid h2 heading styling');
 });
 
+it('passes through image props', () => {
+  const { getByRole } = render(
+    <RichText text={'<img data-test="test123" alt="alt text"></img>'} />,
+  );
+  expect(getByRole('img')).toHaveAttribute('data-test', 'test123');
+  expect(getByRole('img')).toHaveAttribute('alt', 'alt text');
+});
+
+it('sets blank alt when omitted', () => {
```

Comment:
Does this actually happen? :scream: 
I hoped the rich text editor would mandate alt text like MD does.
`alt=""` signals that an image is purely decorative, which we can't really guarantee

Reply:
This was to keep ES Lint happy

Reply:
Yeah I figured, but doesn't answer the question. If it's just to keep ESLint happy and not supposed to happen (i.e. Squidex only ever allows images with alt text), we could throw if it is missing.

Reply:
[we discussed this in call]